tests/test_arrow.py

```diff
@@ -549,6 +583,62 @@ def test_to(self):
         assert arrow_from.to("UTC").datetime == self.expected
         assert arrow_from.to(tz.tzutc()).datetime == self.expected
 
+    # issue #368
+    def test_to_pacific_then_utc(self):
+        result = arrow.Arrow(2018, 11, 4, 1, tzinfo="-08:00").to("US/Pacific").to("UTC")
+        assert result == arrow.Arrow(2018, 11, 4, 9)
+
+    # issue #368
+    def test_to_amsterdam_then_utc(self):
+        result = arrow.Arrow(2016, 10, 30).to("Europe/Amsterdam")
+        assert result.utcoffset() == timedelta(seconds=7200)
+
+    # regression test for #690
+    def test_to_israel_same_offset(self):
+
+        result = arrow.Arrow(2019, 10, 27, 2, 21, 1, tzinfo="+03:00").to("Israel")
+        expected = arrow.Arrow(2019, 10, 27, 1, 21, 1, tzinfo="Israel")
+
+        assert result == expected
+        assert_datetime_equality(result, expected)
```

Comment:
Do you think these checks are these checks redundant?

Reply:
This one probably is.